### ISSUE TITLE
e2e Tests for Federated Daemonsets are not running

### ISSUE THREAD

**Issue by quinton-hoole** [kubernetes/kubernetes#36665]
They need a "Feature:Federation" label, like all the other Federation e2e tests.

  https://github.com/kubernetes/kubernetes/pull/35067

**Comment by dims** [comment]
This needs to be triaged as a release-blocker or not for 1.5 @quinton-hoole

**Comment by dims** [comment]
@mwielgus all issues must be labeled either release blocker or non release blocking by end of day 18 November 2016 PST. (please add the appropriate label or please move it to 1.6) cc @kubernetes/sig-cluster-federation

**Comment by quinton-hoole** [comment]
Ping @mwielgus ?  We really need this in 1.5 in order to release federated daemonsets.

**Comment by quinton-hoole** [comment]
cc @matchstick @nikhiljindal

**Comment by madhusudancs** [comment]
This is a release blocker in the sense that we can't ship Federated DaemonSets without this. It is not a release blocker in the sense that we can ship v1.5 without Federated DaemonSets. So leaving the v1.5 milestone as is and adding a `release-blocker` label for further discussion.

**Comment by nikhiljindal** [comment]
I had fixed this in https://github.com/kubernetes/kubernetes/pull/36630. Daemonset tests should now be running. I cant verify right now because of https://github.com/kubernetes/test-infra/pull/1129.
Will close this once that is merged and we have had a successful run: https://k8s-testgrid.appspot.com/google-federation

**Comment by dims** [comment]
@nikhiljindal https://github.com/kubernetes/test-infra/pull/1129 has merged Can you please check on the runs?

**Comment by nikhiljindal** [comment]
Yes its running now. One of the daemon set tests is failing, but there is a separate issue for that.
